pkg/globalmanager/runtime/worker.go

```diff
@@ -170,6 +180,107 @@ func CreatePodWithTemplate(client kubernetes.Interface, object CommonInterface,
 	return createdPod, nil
 }
 
+//CreateEdgeMeshService creates a kubeedge edgemesh service for an object, and returns an edgemesh service URL.
```

Comment:
Add extra a space after `//`, same with other places.
```suggestion
// CreateEdgeMeshService creates a kubeedge edgemesh service for an object, and returns an edgemesh service URL.
```

Reply:
fix